doc/source/release-notes/Percona-Server-5.5.46-37.5.rst

```diff
@@ -0,0 +1,45 @@
+.. rn:: 5.5.46-37.6
+
+==============================
+ |Percona Server| 5.5.46-37.5
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.5.46-37.5 on November 5th, 2015. Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.46-37.5/>`_ and from the :doc:`Percona Software Repositories </installation>`.
+
+Based on `MySQL 5.5.46 <http://dev.mysql.com/doc/relnotes/mysql/5.5/en/news-5-5-46.html>`_, including all the bug fixes in it, |Percona Server| 5.5.46-37.5 is now the current stable release in the 5.5 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.5.46-37.5 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.5.46-37.5>`_. 
+
+New Features
+============
+
+ |Percona Server| is now available for *Ubuntu* 15.10 (Wily).
+
+Bugs Fixed
+==========
+
+ Manipulating the :variable:`innodb_track_redo_log_now` variable dynamically would crash the server if it was started without :variable:`innodb_track_changed_pages` enabled. This variable is available on debug builds only. Bug fixed :bug:`1368530`.
+
+ A potential crash in handling corrupted tables with :variable:`innodb_corrupt_table_action` ``warn`` or ``salvage`` values has been fixed. Bug fixed :bug:`1426610`.
+
+ If the user had duplicate pid-file options in config files when running ``yum upgrade``, the upgrade would stop with error because it would think it found the duplicate pid while it was the same pid specified twice.  Bug fixed :bug:`1454917`.
+
+ On some filesystems server would not start if :ref:`changed_page_tracking` feature was enabled and :variable:`innodb_flush_method` variable was set to ``O_DIRECT``. Bugs fixed :bug:`1500720` and :bug:`1498891`.
+
+ When :ref:`user_stats` are enabled, after executing any statement of the ``SHOW ...`` family that produces non-empty result, the total number of the sent rows is updated in the :table:`USER_STATISTICS` ``ROWS_FETCHED`` column. Bug fixed :bug:`1510953`.
```

Comment:
When userstats are enabled, executing any statement of the SHOW family with non-empty result, would bump U_S.ROWS_FETCHED column values erroneously